pylint/__init__.py

```diff
@@ -11,17 +11,22 @@
 
 import os
 import sys
+from typing import List, Optional
 
 from pylint.__pkginfo__ import __version__
 
 # pylint: disable=import-outside-toplevel
 
 
-def run_pylint():
+def run_pylint(*, arguments: Optional[List[str]] = None):
```

Comment:
I'm probably a bit late on this one.

I would suggest using `args` or `argv` instead, without the keyword requirement. This is what I usually see in these cases.
```py
def main(argv: Optional[Sequence[str]] = None):
    argv = argv or sys.argv[1:]
    ...
```
Keyword only isn't really necessary here, it's just one argument. Even with more, the first will likely always be `args` / `argv`. More detail (optional) once can always be keyword only later, although even then not strictly necessary.